phobos/README.md

```diff
@@ -0,0 +1,123 @@
+# Install rpc-ceph in the Phobos lab OnMetal 
+
+## Get Access
+
+Request VPN access and a cloud login to the phobos lab from [Jay Payne](mailto:jay.payne@rackspace.com)
+
+## Connect to the VPN
+
+Use your credentials to connect to the phobos VPN via Cisco anyconnect or the openconnect client
+
+## Change your password
+
+Your initial password will be a temporary password like `changeme`.  Login to the [Horizon WebUI](https://172.20.4.10) and change it
+
+## Setup your openrc file
+
+* replace `your.username` with your username
+* replace `yourpassword` with your password
+
+```bash
+cat <<EOF > ~/.phobosrc
```

Comment:
All good - minor note for your own interest, to check out. Try using clouds.yaml - https://docs.openstack.org/python-openstackclient/pike/configuration/index.html
It's really nice for when you want to connect to multiple envs. You may already know about it in which case all good :)

e.g.:
openstack --os-cloud phobos <other things>
Or even: 
alias phobos="openstack --os-cloud phobos"

phobos server create ... ... etc.

The shade plugins can also use the clouds.yaml file, so you wouldn't need to source anything later on.

Reply:
Good call out.  I'll switch from rc file to clouds.yaml here.